avidemux_ru.ts

```diff
@@ -9395,7 +9396,7 @@ I will assume it has and try to continue</source>
         <location line="+3"/>
         <location filename="../../../avidemux_plugins/ADM_videoFilters6/ivtcDupeRemover/ADM_ivtcDupeRemover.cpp" line="+3"/>
         <source>_Frame rate change:</source>
-        <translation>_Изменение частоты кадров:</translation>
+        <translation>_Смена кадровой частоты:</translation>
```

Comment:
Are you sure? This feels really wrong for me (and is very unpopular according to $SEARCH_ENGINE).

Reply:
Will be reverted, but it's not wrong, just another wording — https://ru.wikipedia.org/wiki/%D0%9A%D0%B0%D0%B4%D1%80%D0%BE%D0%B2%D0%B0%D1%8F_%D1%87%D0%B0%D1%81%D1%82%D0%BE%D1%82%D0%B0